README.md

```diff
@@ -18,9 +18,7 @@ Otherwise, just `pip install` it:
 $ pip install airflow-providers-wherobots
 ```
 
-## Usage
-
-### Create a connection
+### Create an http connection
 
 You first need to create a Connection in Airflow. This can be done from
 the UI, or from the command-line. The default Wherobots connection name
```

Comment:
Can it be created from the Wherobots UI or the Airflow UI?

Reply:
Good point. I have updated it, added a link to the Airlfow document